project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts

```diff
@@ -82,159 +84,271 @@ export class UsersViewComponent implements OnInit {
+  //             this.filterData('')
+  //             this.snackBar.open('Activated successfully!')
+  //           }, 1500)
+
+  //           // this.getAllKongUsers()
+  //           // // this.getAllUsers()
+  //           // this.snackBar.open(response.params.errmsg)
+  //         } else {
+  //           this.loaderService.changeLoad.next(false)
+  //           this.snackBar.open('Updat unsuccess!')
+  //         }
+  //       }, _err => this.snackBar.open('Error in active'))
+  //       break
+  //     case 'deactive':
+  //       _.set(user, 'isActive', false)
+  //       _.set(user, 'roles', _.map(_.get($event.row, 'role'), i => i.roleName))
+  //       this.usersService.newUnBlockUserKong(loggedInUserId, user.userId).subscribe(response => {
+  //         if (response) {
+  //           // this.getAllUsers()
+  //           this.snackBar.open(response.params.errmsg)
+  //         }
+  //       }, _err => this.snackBar.open('Error in Active'))
+  //       break
+  //     case 'active':
+  //       _.set(user, 'isActive', true)
+  //       _.set(user, 'isBlocked', false)
+  //       _.set(user, 'roles', _.map(_.get($event.row, 'role'), i => i.roleName))
+  //       this.usersService.deActiveUser(user).subscribe(response => {
+  //         if (response) {
+  //           // this.getAllUsers()
+  //           this.snackBar.open('Updated successfully !')
+  //         }
+  //       })
+  //       break
+  //   }
   // }
-  getAllKongUsers() {
-    const rootOrgId = _.get(this.route.snapshot.parent, 'data.configService.unMappedUser.rootOrg.rootOrgId')
-    this.loaderService.changeLoad.next(true)
-    this.usersService.getAllKongUsers(rootOrgId).subscribe(data => {
-      if (data.result.response.content) {
-        this.userWholeData = data.result.response.content || []
-        this.filter(this.currentFilter ? this.currentFilter : 'active')
-      }
-    })
-  }
+
   raiseTabTelemetry(sub: string, data: WsEvents.ITelemetryTabData) {
     this.events.handleTabTelemetry(sub, data)
   }
-  filter(key: string) {
-    const usersData: any[] = []
-    let index = 0
-    let data: any
-    if (key) {
-      this.currentFilter = key
-      this.data = []
-      switch (key) {
-        case 'active':
-          index = 1
-          data = {
-            index,
-            label: key,
-          }
-          this.raiseTabTelemetry(key, data)
-          this.newKongUser(false)
-          break
-        case 'inactive':
-          index = 2
-          data = {
-            index,
-            label: key,
-          }
-          this.raiseTabTelemetry(key, data)
-          this.newKongUser(true)
-          break
-        case 'blocked':
-          this.data = usersData
-          break
-        default:
-          this.data = usersData
-          break
-      }
-    }
-  }
-  newKongUser(active: boolean) {
-    // const rootOrgId = _.get(this.route.snapshot.parent, 'data.configService.unMappedUser.rootOrg.rootOrgId')
-    const usersData: any[] = []
```

Comment:
Why this whole code has been removed?

Reply:
implemented separate function for Active & Inactive users, this is not needed now.